src/views/mfa-verify/PushForm.js

```diff
@@ -49,6 +63,19 @@ define(['okta'], function (Okta) {
         factorName: this.model.get('factorLabel'),
         deviceName: this.model.get('deviceName')
       });
+
+      var that = this;
+      if (that.enabled) {
+        window.setTimeout(function () {
```

Comment:
I mentioned this yesterday, but if that was the case, then wrap the submit function in a defer. We should not be "clicking on the button to trigger a submit".